modules/consul-cluster/main.tf

```diff
@@ -52,6 +52,20 @@ resource "aws_autoscaling_group" "autoscaling_group" {
     ]
   )
 
+  dynamic "initial_lifecycle_hook" {
```

Comment:
Please update the minimum Terraform version in this file to `>= 0.13.0` to reflect that we're using validations now.

Reply:
done!